fsttest/recorder.go

```diff
@@ -28,12 +29,20 @@ func NewRecorder() *ResponseRecorder {
 
 // Header returns the response headers to mutate within a handler.
 func (r *ResponseRecorder) Header() fsthttp.Header {
-	return r.HeaderMap
+	if !r.headersDone {
+		return r.HeaderMap
+	}
+	// Once the send the headers, return a copy so any changes
+	// are discarded.
+	return r.HeaderMap.Clone()
```

Comment:
```suggestion
	if !r.headersSent {
		return r.HeaderMap
	}
	// Once sent, return a copy so any changes are discarded.
	return r.HeaderMap.Clone()
```
I find the name `headersSent` to be more self-explanatory than `headersDone`, so I suggest changing that throughout.

Reply:
Done.